### PR TITLE
Update EKS installation guide for 0.14.0

### PR DESCRIPTION
**Pull request by ianstanton**
- Update EKS installation guide to cover helm3 based release
- Related to astronomer/issues#1358

### CODE REVIEW

File: v0.14/ee-installation-eks.md [hunk 1]
```diff
@@ -306,32 +279,53 @@ astronomer-registry                  ClusterIP      172.20.100.102   <none>
 
 ```
 
-
 Navigate to your newly created load balancer and copy the DNS name: route and use this to create a new wildcard CNAME record in you DNS. If your base domain is *organization.io* your wildcard record should be *.organization.io* and will route traffic to your ELB using that DNS name.
```
Comment (ianstanton) [inline v0.14/ee-installation-eks.md:283]: This is true if you use a wildcard DNS record. We should have an example of what it looks like if you use individual records.

File: v0.14/ee-installation-eks.md [hunk 2]
```diff
@@ -5,103 +5,74 @@ date: 2018-10-12T00:00:00.000Z
...
 
-All Astronomer services will be tied to a base domain of your choice. You will need the ability to add / edit DNS records under this domain.
+Once created, your Astronomer base domain will be linked to a variety of sub-services that your users will access via the internet to manage, monitor and run Airflow on the platform.
 
-Here are some examples of accessible services when we use the base domain `astro.mydomain.com`:
+For the base domain `astro.mydomain.com`, for example, here are some corresponding URLs that your users would be able to reach: 
 
 * Astronomer UI: `app.astro.mydomain.com`
-* New Airflow Deployments: `deployments.astro.mydomain.com/uniquely-generated-airflow-name`
+* Airflow Deployments: `deployments.astro.mydomain.com/uniquely-generated-airflow-name`
```
Comment (ianstanton) [inline v0.14/ee-installation-eks.md:32]: Example url here should be `deployments.astro.mydomain.com/uniquely-generated-airflow-name/airflow`

**Comment by paolaperaza** [comment]
Nice, thanks @ianstanton - both of those commits look good to me. I think we're ready for review?

**Comment by paolaperaza** [comment]
@ianstanton Did you want to replace the output of `kubectl get pods --namespace <my-namespace>` with an updated set of pods?

**Comment by ianstanton** [comment]
> @ianstanton Did you want to replace the output of `kubectl get pods --namespace <my-namespace>` with an updated set of pods?

Thanks - forgot about that! Will update.

**Comment by paolaperaza** [comment]
Looks good to me, @ianstanton !

**Comment by vishwas-astro** [comment]
Looks great @ianstanton and @paolaperaza. Thanks for doing this.

**Comment by vparekh94** [comment]
We'll need to create similar PRs for the other guides as well :)

**Pull request merged**